platform-modules/content-manager/src/main/java/org/ekstep/content/operation/finalizer/ReviewFinalizer.java

```diff
@@ -255,7 +255,8 @@ private void validateResource(String collectionId){
 				notPublishedList.add(resource.getIdentifier());
 			}
 		}
-		if(CollectionUtils.isNotEmpty(notPublishedList)){
+		Boolean childPublishEnabled = Platform.config.getBoolean("collection.only_published_children");
```

Comment:
@pritha-tarento - you have to check the configuration exists or not and provide default value here.

Reply:
Boolean childPublishEnabled =  Platform.config.hasPath("collection.only_published_children") ? Platform.config.getBoolean("collection.only_published_children") : true;

Reply:
Addressed this for default